### ISSUE TITLE
[new rule] prefer toBe(boolean)

### ISSUE THREAD

**Issue by KevinHerklotz** [jest-community/eslint-plugin-jest#204]
It is not best practice to use `toBeTruthy()` and `toBeFalsy()` ([Explanation](https://vincenttunru.com/toBeTruthy-vs-toBe-true/))
We could introduce a new rule called "prefer-to-be-boolean" (not sure about the name) that works like this:

The following pattern is considered warning:

```js
expect(true).toBeTruthy();
expect(false).toBeFalsy();
```

The following pattern is not warning:

```js
expect(true).toBe(true);
expect(false).toBe(false);
```

What do you think about this rule?

**Comment by macklinu** [comment]
I personally agree with this rule in my experience and think a new rule would be a welcome addition. Not sure about the name either, but we can always touch that up in this issue or in a future pull request. 💃

**Comment by SimenB** [comment]
Yeah, I'll never turn down a rule 🙂 `avoid-truthy-falsy`?

**Comment by noahbenham** [comment]
I'd love to pick this one up.

**Comment by SimenB** [comment]
Go for it!
